### DISCUSSION TITLE
Why do I move down after clicking the picture, and then return to the initial position, there is no change in transparency

### DISCUSSION THREAD

**Discussion by shenhaoguangDayDayUp** [pmndrs/react-spring#1754] [Support]
https://codesandbox.io/s/jolly-feather-dvho1?file=/src/App.tsx

**Comment by joshuaellis** [comment]
Because currently we do not support `exitBeforeEnter` on transition items in stable. It is on a `9.4.0-alpha.0`. However, if you want to do it with a stable package, then position the items absolutely within a container.

**Reply by shenhaoguangDayDayUp** [reply]
thanks for your reply ,but I don't understand what you mean too much, can you give me a demo?
https://codesandbox.io/s/jolly-feather-dvho1?file=/src/App.tsx

**Reply by shenhaoguangDayDayUp** [reply]
> 9.4.0-alpha.0

when I use 9.4.0-alpha.0 but it still doesn't work,what can i do?
